ios/RCTWebRTC/ScreenCapturer.m

```diff
@@ -141,6 +141,13 @@ - (instancetype)initWithDelegate:(__weak id<RTCVideoCapturerDelegate>)delegate {
     return self;
 }
 
+- (void)setConnection:(SocketConnection *)connection {
```

Comment:
Would it make sense to simplify `stopCapture` to just set connection to `nil` now?

Reply:
yeah, makes sense. Will do another commit with the fix.